nixos/modules/virtualisation/virtualbox-image.nix

```diff
@@ -45,10 +45,41 @@ in {
           The file name of the VirtualBox appliance.
         '';
       };
+      params = mkOption {
+        type = with types; attrsOf (oneOf [ str int bool (listOf str) ]);
+        example = {
+          audio = "alsa";
+          rtcuseutc = "on";
+          usb = "off";
```

Comment:
Might make sense to be able to specify `usb = false;` and have that converted to `usb = "off";` on its own (this is obviously a reverse compatible change and can be addressed later if desired).

Reply:
Ah, good point. Let me find out how that is possible.